_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/mentors-open-source-hackillinois/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
+
+# Mentors
+
+Open source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience; these experienced open source contributors and engineers help students get the ball rolling on contributing to open source. 
+
+To initially introduce students to the mentors, we held panels regarding Web/Data and Systems so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
+
+![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/mentors-open-source-hackillinois/panel_speakers.jpg "panel")
+
+Afterward, hacking began! Students began learning the ropes of open source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand, and were prepared to guide students through the initially intimidating process of understanding a codebase and improving it. 
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/mentors-open-source-hackillinois/powershell.jpg "mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize](https://blog.hackillinois.org/spotlight/2017/09/29/open-source-at-hackillinois-rust-cookbook.html) for HackIllinois 2017. This year, he came back with other students from IUPUI and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long term relationships between students and the open source community, and we're super glad that projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [Rustbridge](https://github.com/rustbridge) are still being contributed to and built on.
```

Comment:
> IUPUI

Indiana University–Purdue University Indianapolis (IUPUI)

> long term

long-term

> Rustbridge

RustBridge

> we're super glad that projects like … are still being contributed to and built on

we're super glad that groups from HackIllinois are still working together to contribute to and built on projects like …